docs/topics/secured-mission-intro.adoc

```diff
@@ -10,7 +10,7 @@ The SSO booster project contains:
 = Standing up {RHSSO}
 The `service.sso.yaml` file contains all OpenShift configuration items to stand up a pre-configured {RHSSO} server. The SSO server configuration has been simplified for the sake of this exercise and does provide an out-of-the-box configuration, with pre-configured users and security settings.
 
-IMPORTANT: It is not recommended to use this SSO configuration in production. Specifically, the simplifications made to the booster security configuration impact the ability to use it in a production environment.
+WARNING: The `service.sso.yaml` file contains very long lines, and some text editors, such as link:https://wiki.gnome.org/Apps/Gedit[gedit], may have issues reading this file. It is also not recommended to use this SSO configuration in production. Specifically, the simplifications made to the booster security configuration impact the ability to use it in a production environment.
```

Comment:
So here we conflate two warnings, one that has a very little severity (`gedit` is buggy) and one that has extremely high severity (don't use this config in production). I think this is a bad idea. Especially if the high-severity issue is mentioned _after_ the low-severity issue.